enzyme/Enzyme/MLIR/Interfaces/CloneFunction.cpp

```diff
@@ -232,6 +240,11 @@ FunctionOpInterface CloneFunctionWithReturns(
 
   {
     auto &blk = NewF.getFunctionBody().front();
+    if (width != 1) {
```

Comment:
similarly this seems wrong?

Reply:
this isn't part of this PR anymore, (rebased onto main).